examples/pytorch/fsdp/README.md

```diff
@@ -0,0 +1,7 @@
+# Basic Example for Using PyTorch Fully Sharded Data Parallel mode with Transformer Engine
+
```

Comment:
Add some comment for each of those lines (what does each of the example showcase)?